src/test/java/br/com/openenade/api/regiao/RegiaoServiceTests.java

```diff
@@ -0,0 +1,64 @@
+package br.com.openenade.api.regiao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class RegiaoServiceTests {
+
+    @Autowired
+    private RegiaoService service;
```

Comment:
You can use RegiaoController instead of RegiaoService :).

Reply:
What justifies the use of the Controller instead of a Service?

Reply:
@emanueljoivo using the Controllers methods will cover the workflow of the application operations.